script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -967,18 +967,18 @@ I breathe out a sigh.
 [sha:af533e3d02985cbc9bbfcc10b8749c70124737d0]{
 -- Page 360, Offset 25178.
 -- ……でもまあ、その気楽さに救われた。
-...But I guess in a way I'm saved by her carefree attitude.
+But I guess in a way I'm saved by her carefree attitude.
 }
 [sha:97bbdcbc3dddd0fff238f3267c323724a12dffeb]{
 -- Page 360, Offset 25179.
 -- こっちがいくら深刻に考えたところで、血を吸うか吸わないかはアルクェイドにとってどうでもいい問題みたいだ。
-No matter how seriously I think about it,
+Even if I'm over here pondering it so deeply,
```

Comment:
Feels just slightly off to me, as though his thinking about it has bearing on what she does when I think it's more of just a comparison?
> I'd gotten all caught up thinking about it, but it seems...

Reply:
"Feels just slightly off to me, as though his thinking about it has bearing on what she does"
That's literally what he's trying to say, see the follow-up sentence.

Reply:
Not sure I agree with that interpretation still. I read the second one as "it's none of my business, I shouldn't be thinking too deeply / prying into it" rather than trying to influence arc.

Reply:
The impetus on the sentences is こっちがいくら
So basically, he himself is the focus. To illustrate with emphasis using the old sentences

No matter how seriously >>I<< think about it...
the question of whether to drink blood or not doesn't seem to bother >>Arcueid<< in the least.

It's a comparison of their own attitudes. Maybe I should have changed the whole sentence:

Even if %{i}I'm%{/i} over here pondering it so deeply,
 %{i}Arcueid%{/i} doesn't seem to be bothered in the slightest about question of whether to drink blood or not.

Reply:
> It's a comparison of their own attitudes.

I think we might actually be on the same page about this? Shiki is saying "damn, I'm really over here thinking hard about this but she just doesn't care at all". The `Even if` changes the tone of the sentence slightly to me such that it gives the impression instead of "I'm over here thinking hard about this, _but despite that_, she doesn't care at all.

Reply:
Maybe "Even though..." get that nuance across better then...?

Reply:
Yeah, I think that's a bit better. If we can avoid `even` that just does away with the possibility to take it the wrong way imo, hence either the original suggestion or something like
> Regardless of how much thought I dedicate to it, I get the impression that the question of whether she should drink blood or not doesn't weigh on Arcueid's mind at all.

if you want to keep the "doesn't matter what I think" more visible

Reply:
Sure, let's go with that.